core/modules/czar/MessageTable.cc

```diff
@@ -68,6 +70,19 @@ MessageTable::MessageTable(std::string const& tableName,
       _sqlConn(std::make_shared<sql::SqlConnection>(resultConfig)) {
 }
 
+// Create the table, do not lock
+void
+MessageTable::create() {
```

Comment:
Apart from minor changes in the logged messages the only difference between MessageTable::create() and MessageTable::lock() is in the query template string. Could this be just one method with the optional parameter "bool lock=true" instead? Otherwise there is a great (may be not so great) deal of code duplication with the introduction of the additional method.